x-pack/solutions/security/plugins/security_solution/server/lib/siem_migrations/rules/api/util/tags.ts

```diff
@@ -5,9 +5,10 @@
  * 2.0.
  */
 
+import type { OriginalDashboard } from '../../../../../../common/siem_migrations/model/dashboard_migration.gen';
 import type { OriginalRule } from '../../../../../../common/siem_migrations/model/rule_migration.gen';
 
-export const getVendorTag = (originalRule: OriginalRule): string => {
+export const getVendorTag = (originalRule: OriginalRule | OriginalDashboard): string => {
```

Comment:
Maybe it's better to receive the `vendor` directly, like:

```suggestion
import type { SiemMigrationVendor } from '../../../../../../common/siem_migrations/types';

export const getVendorTag = (vendor: SiemMigrationVendor): string => {
```